src/sail_json_backend/json.ml

```diff
@@ -843,7 +894,11 @@ let defs { defs; _ } =
     List.sort (fun l r -> String.compare (List.hd (List.tl l)) (List.hd (List.tl r))) key_mnemonic_map
   in
   print_endline
-    (String.concat ",\n" (List.map (fun a -> json_of_instruction (List.hd a) (List.tl a)) key_mnemonic_sorted));
+    (String.concat ",\n" (List.map (fun a ->
+        let key = List.hd a in
+        let value = List.tl a in
+        json_of_instruction key value
+    ) key_mnemonic_sorted));  
```

Comment:
Is this a formatting change suggested by `dune fmt`?
It looks functionally identical. Is this needed?